### ISSUE TITLE
Improve Add Block button icon

### ISSUE THREAD

**Issue by etoledom** [wordpress-mobile/gutenberg-mobile#1457]
The current `Add Block` icon is the [plus-alt DashIcon](https://developer.wordpress.org/resource/dashicons/#plus-alt). The problem with this icon is that it's not as think as required by design, so we worked around by adding a margin of the same color of the icon itself.

This brings some issue as noted here: https://github.com/wordpress-mobile/WordPress-iOS/pull/12691#issuecomment-542221433

We should replace this icon with a proper one with the desired design.

**Comment by iamthomasbishop** [comment]
> We should replace this icon with a proper one with the desired design.

@etoledom Agreed. I've never been a big fan of the Dashicons version of this icon. My proposal originally was to use the Material variations for the filled and outlined variations. So maybe now is the time to include those for mobile consumption. You can download those as svg icons [here](https://zeroheight.com/226850145/p/345d49-icons/t/15c007).

(To download, click on the desired icon (`add_circle_24px` and `add_circle_outline_24px`, then on the right side click on the "slice"/assets tab, and you can download the svg directly from there.)

One thing to note: I _think_ Dashicons are designed on a 20px grid, while Material are designed for a 24px grid. If possible, let's make sure the 24px icon is honored esp for the Appender icon.
